hack/windows/Prepare-Node.ps1

```diff
@@ -82,21 +90,24 @@ mkdir -force C:\var\log\kubelet
 mkdir -force C:\var\lib\kubelet\etc\kubernetes
 mkdir -force C:\etc\kubernetes\pki
 New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C:\etc\kubernetes\pki\
-
+{{CONTAINER_RUNTIME}}
```

Comment:
what does this mean? only a variable in a line?

Reply:
removed this line